### PR TITLE
Adding draft of signing doc

### PR DESCRIPTION
**Pull request by ltagliaferri**
The main purpose of this PR is to add docs on Enforce Signing. Please test and edit the tutorial. Additionally, I edited the cloud account association doc based on going through that workflow to set up Enforce Signing.

I also did some left nav bar clean up and light edits elsewhere. 

Signed-off-by: ltagliaferri

### CODE REVIEW

**Comment by ltagliaferri** [comment]
Need to add beta / preview flag

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+```
+
+You can also verify with a wildcard certificate identity, as in the following command.
+
+```sh
+eval $(chainctl sigstore env $ENFORCE_CA_NAME)
+cosign verify $IMAGE --certificate-identity-regexp=".*"
+```
+
+After running this command, you should receive output that returns that the Cosign claims and certificate were verified.
```
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md:196]: Is it the same / similar output for both of these `cosign verify` commands? This output looks like what I received from running the first command, but as I mentioned the second one didn't work for me.

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+```
+Verification for $IMAGE --
+The following checks were performed on each of these signatures:
+  - The cosign claims were validated
+  - The code-signing certificate was verified using trusted certificate authority certificates
+
+[{"critical":{"identity":{"docker-reference":"$IMAGE"},"image":{"docker-manifest-digest":"sha256:77...9"},"type":"cosign container image signature"},"optional":{"...":"https://auth.chainguard.dev/","Issuer":"https://auth.chainguard.dev/","..."},"Subject":"$EMAIL"}}]
+```
+
+Congratulations! You have signed and verified a software artifact with Chainguard Enforce Signing!
```
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md:207]: I might tone down the congratulations here. Maybe just "This output indicates that software artifact's signature was successfully verified."

**Review by SharpRake — COMMENTED** [review]
This is great! the tech test worked like a charm except for the last command, but I may not have the latest version of cosign installed so that could be it.

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+To set up a Google CA Service, you'll first need an instance of the GCP CA Service in your GCP Project. You can enable the CA service by running the following command. This may take a few minutes.
+
+```sh
+gcloud services enable privateca.googleapis.com
+```
+
+You'll need create a new CA Pool (alternately, you can use an existing one if you have one set up). For demonstration purposes, we'll use the `devops` tier. Replace `$POOL_NAME` with the name you would like to use for your CA Pool. 
+
+```sh
```
[the remark this answers is not in the document]
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md:47]: I added this line: 

```
Depending on your configuration, you may need to pass the location of where you want this CA Pool, with the `--location=` flag. 
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+eval $(chainctl sigstore env $ENFORCE_CA_NAME)
+cosign verify $IMAGE --certificate-identity=$EMAIL
+```
+
+You can also verify with a wildcard certificate identity, as in the following command.
+
+```sh
+eval $(chainctl sigstore env $ENFORCE_CA_NAME)
+cosign verify $IMAGE --certificate-identity-regexp=".*"
```
[the remark this answers is not in the document]
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md:193]: Yeah it needs to be the newest Cosign pulled from GH

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md (hunk 1 above)
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-signing/getting-started-chainguard-enforce-signing.md:196]: It should be the same output either way

**Review by SharpRake — APPROVED** [review]

**Pull request merged**